examples/zephyr/location/pytest/test_sample.py

```diff
@@ -25,19 +23,11 @@ async def test_location(shell: Shell, dut: DeviceAdapter, device):
 
     dut.readlines_until(regex=".*Golioth CoAP client connected", timeout=90.0)
 
-    # Verify position
-
-    pattern = re.compile(r".* (?P<lon>\d+\.\d+) (?P<lat>\d+\.\d+) \((?P<acc>\d+)\)")
-    Position = namedtuple('Position', ('lon', 'lat', 'acc'))
-
-    positions = []
-    for _ in range(3):
-        lines = dut.readlines_until(regex=pattern, timeout=30.0)
-        m = pattern.search(lines[-1])
-        pos = Position(*[float(m[p]) for p in ['lon', 'lat', 'acc']])
-        positions.append(pos)
-
-    for pos in positions:
-        assert pos.lon == pytest.approx(50.663974800, 1e-3)
-        assert pos.lat == pytest.approx(17.942322850, 1e-3)
-        assert pos.acc < 2000
+    # Verify network info data
+    count = 0
+    async with contextlib.aclosing(device.stream.iter()) as stream_iter:
+        async for value in stream_iter:
+            assert "cell" in value["data"] or "wifi" in value["data"]
+            count += 1
+            if count == 3:
+                break
```

Comment:
I think we could improve this (and perhaps even change the example to some degree), but each iteration of the test (cell, wifi, and both) passed on at least one of the platforms on the most recent run, and the failures look related to websocket issues with stream, so I'm going to move forward with it for the time-being.